<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>760</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Записная книга 2.0</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Записные книги</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="handleWidth">
           <number>13</number>
          </property>
          <property name="childrenCollapsible">
           <bool>false</bool>
          </property>
          <widget class="QWidget" name="">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="sizeConstraint">
               <enum>QLayout::SetMinAndMaxSize</enum>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>15</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Быстрый поиск</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>Arial</family>
                 </font>
                </property>
                <property name="text">
                 <string>Введите номер записной книги:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="text">
                 <string>Поиск</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="vl_cancel_button"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTableView" name="tableView">
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>25</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetMaximumSize</enum>
            </property>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="sizeConstraint">
               <enum>QLayout::SetNoConstraint</enum>
              </property>
              <item>
               <widget class="QLabel" name="label">
                <property name="font">
                 <font>
                  <family>Arial</family>
                 </font>
                </property>
                <property name="text">
                 <string>Телефонные номера владельца</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableView" name="tableView_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                  <horstretch>20</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="sizeConstraint">
               <enum>QLayout::SetNoConstraint</enum>
              </property>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="font">
                 <font>
                  <family>Arial</family>
                 </font>
                </property>
                <property name="text">
                 <string>Список контактов</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableView" name="tableView_3">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="MinimumExpanding">
                  <horstretch>120</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="selectionBehavior">
                 <enum>QAbstractItemView::SelectRows</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::MinimumExpanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>814</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="action_add"/>
   </widget>
   <addaction name="menufile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_add"/>
   <addaction name="separator"/>
   <addaction name="action_delete"/>
   <addaction name="action_update"/>
   <addaction name="action_analysis"/>
   <addaction name="action_search"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_add">
   <property name="text">
    <string>Добавить пользователя</string>
   </property>
   <property name="toolTip">
    <string>Добавление нового пользователя</string>
   </property>
  </action>
  <action name="action_delete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
   <property name="toolTip">
    <string>Удалить выбранный объект</string>
   </property>
  </action>
  <action name="action_update">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Редактировать</string>
   </property>
   <property name="toolTip">
    <string>Редактировать выбранную запись</string>
   </property>
  </action>
  <action name="action_analysis">
   <property name="text">
    <string>Анализ</string>
   </property>
  </action>
  <action name="action_search">
   <property name="text">
    <string>Расширенный поиск</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
